bluetooth-proxy/m5stack-atomS3.yaml

```
@@ -0,0 +1,43 @@
+substitutions:
+  name: atom-bluetooth-proxy
+  friendly_name: Bluetooth Proxy
+
+esphome:
+  name: ${name}
+  friendly_name: ${friendly_name}
+  name_add_mac_suffix: true
+  project:
+    name: esphome.bluetooth-proxy
+    version: "1.0"
+
+esp32:
+  board: m5stack-atoms3
+  variant: esp32s3
+  framework:
+    type: esp-idf
+
+wifi:
+  ap:
+
+api:
+logger:
+ota:
+improv_serial:
```

Needed for upcoming (June) ESPHome release.
```suggestion
improv_serial:

ota:
  - platform: esphome
```